.prettierignore

```diff
@@ -13,7 +13,6 @@ lerna.json
 .github/actions/next-stats-action/.work
 .github/actions/issue-validator/index.mjs
 .github/actions/issue-labeler/lib/index.js
-packages/next-swc/crates/**/*
```

Comment:
When we revert this change, the PR should pass?

Reply:
probably, just did that

also tried to update the prettier PR, if that passes now I'll merge it first, otherwise I'll merge this PR first